home/WEB-INF/jsp/ontology/term-view.jsp

```diff
@@ -107,6 +107,15 @@
                              data-indexer=${UiIndexerConfig.ChebiPhenotypeIndexer.typeName}
                 ></div>
             </z:section>
+            <z:section title="Phenotype affecting" appendedText="${term.termName}" show="${true}">
+                <div class="__react-root" id="ChebiPhenotypeTable"
+                     data-term-id="${term.zdbID}"
+                     data-direct-annotation-only="true"
+                     data-eqe="true"
```

Comment:
What does eqe stand for?

Reply:
EQE = Entity - Quality - Entity
That is the general combination for a phenotype annotation.